datadriven.md

```diff
@@ -0,0 +1,57 @@
+- Learn how to manipulate and clean large datasets, and how to assertively communicate when they have an idea for a use-case on the company's available data
+- Give them a wide view on different data related subjects and fields, so they will later be able to find further tutorials and specialize in these areas on their own
+- Make them able to figure out where to get external datasets for their projects and know how to use them 
+- Hear about use-cases and solutions from data-driven companies (Wallmart, Target, Amazon, Netflix, Spotify, etc.) to have a better idea on how data can be used for profit
+- Even if they do not chose to be a Data Scientist / Data Engineer, they should still be able to help their company with their large amounts of data or go for a Full Stack Python developer career path
+
+At the end of the course the students will implement their very own Movie recommendation system based on the combination of the IMDB, MovieLens and multiple, manually crawled datasets. 
+
+## Prerequisites to take the course
+
+Please only take this course if you are already familiar with the following techniques:
+- Have strong full stack developer skills (HTML, JS, CSS,  REST APIs: XML and JSON)
+- Have learned about either SQL or NoSQL technologies 
+- Familiar with Office Excel (or similar spreadsheet software)
+- Have a **basic** understanding of statistics, probability, linear algebra (you know what vectors and matrices are) - we will not dwell into the math deeply here, but it will be required later in this field to have a deeper understanding. You can always (re)learn these concepts from KhanAcademy!
+- Have at least 8GB of RAM and 10GB free disk space. Any OS is fine, because Python runs on pretty much everything. You can chose your own IDE (or Notebook) for Python
+
+We will use Python for almost every task. However, if you already know C++ or JAVA it will be much easier for you to find a job, as some Deep Learning libraries and Big Data technologies rely on these programming languages (JAVA is a good choice for Cloudera's current solutions). 
+
+The main difference between this class and the others is that there are will be no user stories, and most of the tasks will be required to be done by everyone, instead working together in parallel as a team, to ensure everyone learns the same things. Some of the tasks might not make sense first, but it will become clear by the end of the course why we are doing them. 
+
+Please note that as a Data Scientist / Data Engineer, the amount of possible technologies/models/domain knowledge to learn is **infinite**. 
+No one knows everything in this field, so bright people work together to come up with solutions for the companies. 
+For this reason, you will not feel like you know everything at the end of the course - instead you will end up realizing how much more there is to learn. This is normal, in fact, this is how everyone in the field with experience constantly feels. 
```

Comment:
I think this is true for each language or technology

Reply:
I agree, but I want to make sure they don't feel bad after realizing they were only scratching the surface.